HipercowApiUnitTests/Tools/ClusterHandleCacheTests.cs

```diff
@@ -25,7 +25,7 @@ public void CachedScheduler_Works()
             ClusterHandleCache chc = new ClusterHandleCache(mockSchedulerFactory.Object);
             IScheduler? sch1 = chc.GetClusterHandle("wpia-hn");
             IScheduler? sch2 = chc.GetClusterHandle("wpia-hn");
-            Assert.NotNull(sch1);
+            Assert.Equal(sch1, mockScheduler.Object);
```

Comment:
```suggestion
            mockSchedulerFactory.Verify(x => x.NewScheduler(), Times.Once());
            Assert.Equal(sch1, mockScheduler.Object);
```
I still think this would be valid, since it could have called the mocked method twice and got the same scheduler back both times. .